MegaQuests/src/main/java/com/elmer/megaquests/listeners/QuestGUIListener.java

```diff
@@ -6,10 +6,14 @@
 import org.bukkit.event.inventory.InventoryClickEvent;
 
 public class QuestGUIListener implements Listener {
+
     @EventHandler
-    public void onInventoryClick(InventoryClickEvent e){
-        if(ChatColor.translateAlternateColorCodes('&', e.getView().getTitle()).equals(ChatColor.YELLOW.toString() + ChatColor.BOLD + "Quests") && e.getCurrentItem() != null){
-            e.setCancelled(true);
+    public void onInventoryClick(InventoryClickEvent event) {
+        boolean isSameItem = ChatColor.translateAlternateColorCodes('&', event.getView().getTitle())
+                .equals(ChatColor.YELLOW.toString() + ChatColor.BOLD + "Quests");
+
+        if (event.getCurrentItem() != null && isSameItem) {

```

Comment:
If one check is too long, it's better to put it in a sound variable, whose name will tell you what it does and then pass it into the condition.